packages/serenity/src/command/CommandManager.ts

```diff
@@ -0,0 +1,49 @@
+import { readdirSync } from 'node:fs';
+import { fileURLToPath, URL } from 'node:url';
+import type { Player, Serenity } from '../index.js';
+import type { Command } from './Command.js';
+
+class CommandManager {
+	private readonly serenity: Serenity;
+	private commands: Map<string, Command> = new Map();
+
+	public constructor(serenity: Serenity) {
+		this.serenity = serenity;
+    
+     void this.start();
+	}
+
+	public async start(): Promise<void> {
+		const commands = readdirSync(fileURLToPath(new URL('defaults', import.meta.url)))
```

Comment:
This wont work in long term. Please create an array list of default commands

Reply:
What do you suggest? I'm thinking of keeping the command classes similar to how the handlers are, for example